### PR TITLE
pacmc: Improve license name, fix extract_dir

### PR DESCRIPTION
**Pull request by bluefireoly**

### CODE REVIEW

**Comment by bluefireoly** [comment]
The `extract_dir` now explicitly contains the package version, I will change this in the future, when the released archive on GitHub doesn't include the version in that directory anymore.

**Comment by bluefireoly** [comment]
Okay so what did this mess of commits change?
 - added `notes`
 - fixed `extract_dir` by including it in `autoupdate`
 - switch to version `0.1.2`

File: bucket/pacmc.json [hunk 1]
```diff
@@ -1,12 +1,13 @@
 {
-    "version": "0.1.1",
+    "version": "0.1.2",
     "description": "An easy to use package manager for Fabric Minecraft mods.",
-    "license": "GPL-3.0",
-    "url": "https://github.com/bluefireoly/pacmc/releases/download/0.1.1/pacmc-0.1.1.zip#/pacmc.zip",
-    "extract_dir": "pacmc-$version",
+    "license": "GPLv3 or later",
```
Comment (Calinou) [inline bucket/pacmc.json:4]:
```suggestion
    "license": "GPL-3.0-or-later",
```

**Review by Calinou — APPROVED** [review]
Thanks!

File: bucket/pacmc.json [hunk 2]
```diff
@@ -17,6 +18,6 @@
             "java/openjdk"
         ]
     },
-    "homepage": "https://github.com/bluefireoly/pacmc"
+    "homepage": "https://github.com/bluefireoly/pacmc",
+    "notes": "Thank you for installing pacmc, run the 'pacmc' command to get started."
```
Comment (Calinou) [inline bucket/pacmc.json:22]: We assume that people know how to run applications they just installed using Scoop. Scoop will print messages about binaries being added to the `PATH` or in the Start Menu.

```suggestion
```

File: bucket/pacmc.json [hunk 3]
```diff
@@ -17,6 +18,6 @@
             "java/openjdk"
         ]
     },
-    "homepage": "https://github.com/bluefireoly/pacmc"
+    "homepage": "https://github.com/bluefireoly/pacmc",
```
Comment (Calinou) [inline bucket/pacmc.json:21]:
```suggestion
    "homepage": "https://github.com/bluefireoly/pacmc"
```

File: bucket/pacmc.json [hunk 4]
```diff
@@ -1,12 +1,13 @@
 {
-    "version": "0.1.1",
+    "version": "0.1.2",
     "description": "An easy to use package manager for Fabric Minecraft mods.",
```
Comment (Calinou) [inline bucket/pacmc.json:3]:
```suggestion
    "description": "An easy to use package manager for Fabric Minecraft mods",
```

**Comment by bluefireoly** [comment]
Is there a mistake in my `autoupdate` and `checkver` "logic"? Because there has been a new release of pacmc on GitHub 2 days ago, but the update still has not been committed to scoop-games by the bot.
Am I overseeing something here?

**Comment by bluefireoly** [comment]
Ah I see, thank you for doing this right away <3

**Comment by bluefireoly** [comment]
> I fixed this in 08ff686.

you forgot to update the `extract_dir` in this manual update

**Comment by Calinou** [comment]
Fixed in https://github.com/Calinou/scoop-games/commit/604cc42478e41106782dc726907334c7cb5c4912.

**Pull request merged**